iib/web/models.py

```diff
@@ -882,13 +908,15 @@ def from_json(cls, kwargs, batch=None):
                 'organization',
                 'bundles',
                 'distribution_scope',
+                'deprecation_list',
             ],
             batch=batch,
         )
 
-        request_kwargs['bundles'] = [
-            Image.get_or_create(pull_specification=item) for item in bundles
-        ]
+        for key in ('bundles', 'deprecation_list'):
```

Comment:
Why is at line #876 the same for cycle? Could we put it together?

Reply:
I kept them separate because the #876 is for validating the input and this one creates entries into the database. So it made sense to complete all the validation before creating entries into the database.

Reply:
Can I ask you for comments there?

Reply:
@yashvardhannanavati Could you add a comment here?